app/parse.py

```diff
@@ -8,8 +18,62 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[tag.text for tag in quote_soup.select(".tag")]
+    )
+
+
+def get_next_pages(page_soup: BeautifulSoup) -> str:
```

Comment:
You are getting 1 next page. So better to have `get_next_page` naming, without `s`